var/spack/repos/builtin/packages/py-requests/package.py

```diff
@@ -23,6 +23,8 @@ class PyRequests(PythonPackage):
         'requests.packages.urllib3.contrib._securetransport'
     ]
 
+    version('2.24.0', sha256='b3559a131db72c33ee969480840fff4bb6dd111de7dd27c8ee1f820f4f00231b')
+    version('2.23.0', sha256='b3f43d496c6daba4493e7c431722aeb7dbc6288f52a6e04e7b6023b0247817e6')
     version('2.22.0', sha256='11e007a8a2aa0323f5a921e9e6a2d7e4e67d9877e85773fba9ba6419025cbeb4')
     version('2.21.0', sha256='502a824f31acdacb3a35b6690b5fbf0bc41d63a24a45c4004352b0242707598e')
     version('2.14.2', sha256='a274abba399a23e8713ffd2b5706535ae280ebe2b8069ee6a941cb089440d153')
```

Comment:
It looks like some of the dependency versions have changed. Can you update them?
```python
requires = [                                                                    
    'chardet>=3.0.2,<4',                                                        
    'idna>=2.5,<3',                                                             
    'urllib3>=1.21.1,<1.26,!=1.25.0,!=1.25.1',                                  
    'certifi>=2017.4.17'                                                        
]   
```